docs/examples/009-subsetting.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<p>Maybe a transition sentence just before this part, e.g. "In addition to global grid capabilities, UXarray can also provide subsetting functionalities." </p><p> </p><p>Or, maybe crate a new title such as "Subsetting" the same level as the "Global Grid" and put all those sections such as "Bounding Box" under it? Such sections through the end of this NB does not emphasize the <code>.subset</code> namespace, maybe this new "Subsetting" title can briefly describes that, mentioning all the subsetting examples below will use it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/616/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>